### PR TITLE
17744 dashboard va benefits disability

### PR DESCRIPTION
**Pull request by tlei123**
## Description
Update Dashboard Find VA Benefits Disability section.  (See [Issue 17744](/department-of-veterans-affairs/vets.gov-team/issues/17744) for details.)

## Testing done
Pure content/data changes verified on webpage.
[No logic-/functional-code changes; no unit-testing needed]

## Screenshots
Screen Shot 2019-03-21 at 9 48 41 AM

## Acceptance criteria
Under Disability Benefits section:
- [ ] "Find out how to file a claim online, by mail, or in person." no longer on the list of FAQ links.
- [ ] CTA button labelled "Find out how to file a claim" and linked to https://www.va.gov/disability/how-to-file-claim/

## Definition of done
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/platform/site-wide/va-footer/helpers.jsx [hunk 1]
```diff
@@ -20,19 +20,20 @@ export const FOOTER_EVENTS = {
...
+    ) : null}
+    {link.href ? (
+      <a href={link.href} onClick={captureEvent} target={link.target}>
+        {link.title}
+      </a>
+    ) : (
+      <span className="va-footer-link-text">{link.title}</span>
+    )}
+  </>
+);
```
Comment (ncksllvn) [inline src/platform/site-wide/va-footer/helpers.jsx]: Fragments! 😮 🎉

**Review by ncksllvn — APPROVED** [review]
LGTM 👍

**Comment by ncksllvn** [comment]
FYI that `master` has an issue that will need to be resolved before can merge this. We'll have to merge in latest master to this branch at that point. Great work though!

File: src/platform/site-wide/va-footer/helpers.jsx (hunk 1 above)
Comment (erikphansen) [reply src/platform/site-wide/va-footer/helpers.jsx]: Oops, your PR from yesterday didn't get merged? I'm fine with this change being a part of this PR and killing the other PR that had this change 😀

**Review by erikphansen — APPROVED** [review]

File: src/platform/site-wide/va-footer/helpers.jsx (hunk 1 above)
Comment (tlei123) [reply src/platform/site-wide/va-footer/helpers.jsx]: @erikphansen , which "other" PR?

Comment (erikphansen) [reply src/platform/site-wide/va-footer/helpers.jsx]: yeah that's the PR. It makes this same change. so I figured this branch had that work in it as well.

Comment (tlei123) [reply src/platform/site-wide/va-footer/helpers.jsx]: Update: the other branch is now Merged into master (the CI build-check passed once I sync'ed the branch up with master 15-20 min ago and pushed that up)...
So yes, the change is here now too 'cuz I just merged master into it a sec ago.
Just waiting for the check below to pass now.

**Pull request merged**